scripts/preload_models.py

```diff
@@ -36,6 +40,19 @@
 transformer = CLIPTextModel.from_pretrained(version)
 print('...success')
 
+
+print('preloading Stable Diffusion model file...', end='')
+sd1_4 = Path(hf_hub_download("CompVis/stable-diffusion-v-1-4-original", "sd-v1-4.ckpt"))
+# inpainting = Path(hf_hub_download("runwayml/stable-diffusion-inpainting", "sd-v1-5-inpainting.ckpt"))
+# sd1_5 = Path(hf_hub_download("runwayml/stable-diffusion-v1-5", "v1-5-pruned.ckpt"))
+
+model_dest = Path('models/ldm/stable-diffusion-v1/model.ckpt')
```

Comment:
When I tried to run the script in an environment where `model.ckpt` already existed in the destination directory, I got this crash:

~~~
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /u/lstein/projects/SD/InvokeAI/scripts/preload_models.py:51 in <module>                          │
│                                                                                                  │
│    48                                                                                            │
│    49 model_dest = Path('models/ldm/stable-diffusion-v1/model.ckpt')                             │
│    50 model_dest.parent.mkdir(parents=True, exist_ok=True)                                       │
│ ❱  51 sd1_4.resolve(strict=True).link_to(model_dest)                                             │
│    52                                                                                            │
│    53 print('...success')                                                                        │
│    54                                                                                            │
│                                                                                                  │
│ /u/lstein/.conda/envs/invokeai/lib/python3.9/pathlib.py:1415 in link_to                          │
│                                                                                                  │
│   1412 │   │   matches that of os.link.                                                          │
│   1413 │   │                                                                                     │
│   1414 │   │   """                                                                               │
│ ❱ 1415 │   │   self._accessor.link_to(self, target)                                              │
│   1416 │                                                                                         │
│   1417 │   # Convenience functions for querying the stat results                                 │
│   1418                                                                                           │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
FileExistsError: [Errno 17] File exists: 
'/u/lstein/.cache/huggingface/hub/models--CompVis--stable-diffusion-v-1-4-original/blobs/fe4efff1e174c627256e44ec2991ba279b3816e364b49f9be2abc0b3ff3f8556' -> 
'models/ldm/stable-diffusion-v1/model.ckpt'
~~~

Ideally, the script should check whether `model.ckpt` is already installed, and if so ask whether the user wants to overwrite it. Or just print the message "model.ckpt already installed; nothing to do"

Reply:
I certainly _thought_ I'd run this more than once and it didn't trip over its previous output, but clearly I need to check that again!